backend/src/routes/projectRouter.ts

```diff
@@ -134,6 +134,17 @@ projects.get("/v1/project_lookup/:name", async (req, res, next) => {
     next(err);
   }
 });
+//search projects by name and description
+projects.get("/v1/project_search/:search", async (req, res, next) => {
```

Comment:
nit: probably best to do `:search` path parameter as a query parameter instead in case the user submits strings with spaces. (spaces is easier to encode as a query parameter); if the user searches for `project name / desc`, slash is parsed as part of path